tableauserverclient/models/task_item.py

```diff
@@ -0,0 +1,38 @@
+import xml.etree.ElementTree as ET
+from .. import NAMESPACE
+from .schedule_item import ScheduleItem
+
+
+class TaskItem(object):
+    def __init__(self, id_, task_type, priority, consecutive_failed_count=0, schedule_id=None):
+        self.id = id_
+        self.task_type = task_type
+        self.priority = priority
+        self.consecutive_failed_count = consecutive_failed_count
+        self.schedule_id = schedule_id
+
+    def __repr__(self):
+        return "<Task#{id} {task_type} pri({priority}) failed({consecutive_failed_count}) schedule_id({" \
+               "schedule_id})>".format(
+            **self.__dict__)
+
+    @classmethod
+    def from_response(cls, xml):
+        parsed_response = ET.fromstring(xml)
+        all_tasks_xml = parsed_response.findall('.//t:task', namespaces=NAMESPACE)
+
+        all_tasks = (TaskItem._parse_task(x) for x in all_tasks_xml)
+
+        return list(all_tasks)
+
+    @classmethod
+    def _parse_task(cls, element):
```

Comment:
On the other Item classes we call this method `_parse_element` and I'll argue for foolish consistency :)